### ISSUE TITLE
[BUG] We could not parse the JSON body of your request

### ISSUE THREAD

**Issue by edshamis** [0xacx/chatGPT-shell-cli#104]
The command I run:
~/repos/bin/chatGPT-shell-cli/chatgpt.sh -m gpt-4  --init-prompt-from-file ~pr/tutor

The init file is json:
https://raw.githubusercontent.com/JushBJJ/Mr.-Ranedeer-AI-Tutor/main/Mr_Ranedeer.json

The full error:
Your request to Open AI API failed: invalid_request_error
We could not parse the JSON body of your request. (HINT: This likely means you aren't using your HTTP library correctly. The OpenAI API expects a JSON payload, but what was sent was not valid JSON. If you have trouble figuring out how to fix this, please contact us through our help center at help.openai.com.)

**Comment by 0xacx** [comment]
Thanks for reporting!

**Comment by 0xacx** [comment]
This is resolved now, after merging #87 and #106 

```
$ ./chatgpt.sh --init-prompt-from-file tutor.json 
Welcome to chatgpt. You can quit with 'exit' or 'q'.

Enter a prompt:
What are you?

             
chatgpt 

  I am an AI tutor designed to assist students in their learning journey by   
  providing personalized lesson plans and educational support.                


Enter a prompt:
What is your name?

             
chatgpt 

  My name is Mr. Ranedeer. How can I assist you today?                        
```
